Python/Product/Debugger/Debugger/DebugEngine/AD7Engine.cs

```diff
@@ -536,6 +539,10 @@ public sealed class AD7Engine : IDebugEngine2, IDebugEngineLaunch2, IDebugProgra
                 return VSConstants.E_NOTIMPL;
             }
 
+            if (!ExperimentalOptions.UseVsCodeDebugger && !ExperimentalOptions.DontShowUseVsCodeDebuggerInfoBar) {
+                AddInfoBar();
```

Comment:
I think you'll end up with multiple info bars stack on top of each other if you do multiple debugging sessions and the user doesn't dismiss them.